rc/lsp.kak

```diff
@@ -162,6 +165,10 @@ Capture groups must be:
     4: optional message
 } regex lsp_location_format ^\h*\K([^:\n]+):(\d+)\b(?::(\d+)\b)?(?::([^\n]+))
 
+declare-option -docstring "Number of indent levels to skip" int lsp_indent_guides_skip 0
```

Comment:
The purpose of indent guides is so we can see more easily which lines belong together.
At level 0 we already have such a guide, namely the edge of the terminal window.
These two vertical lines don't look good here and I think that's a common pattern.
Also, in most languages it's very obvious if we are inside a function or not (usually we are).

So I think the default should be 1.

Reply:
Agree, changed to 1